prisma/schema.prisma

```diff
@@ -68,11 +68,11 @@ model workflow {
   editor              String?
```

Comment:
Did you connect with Seby about adding both user's displayName as well as user's unique id ?